config/initializers/override_scoped_search_escaping.rb

```diff
@@ -3,8 +3,22 @@
 # Monkeypatching the query parser in order to escape '%' and '_' from searches
 # Code copied from https://github.com/wvanbergen/scoped_search/blob/master/lib/scoped_search/query_builder.rb
 module ScopedSearch
+  OPERATORS = { '=' => :eq, '<>' => :ne, 'LIKE' => :like, 'ILIKE' => :like,
+                'NOT LIKE' => :unlike, 'NOT ILIKE' => :unlike,
+                '>' => :gt, '<' => :lt, '<=' => :lte, '>=' => :gte,
+                'IN' => :in, 'NOT IN' => :notin }.freeze
+
   class QueryBuilder
     alias preprocess_parameters_old preprocess_parameters
+
+    def self.preprocess_parameters(definition, field, operator, value)
```

Comment:
Please add some explanation why this method has been created, in which cases and how it should be used.

Reply:
Okay, just added it 🙂